tee-worker/ts-tests/integration-tests/dr_vc.test.ts

```diff
@@ -147,30 +148,58 @@ describe('Test Vc (direct request)', function () {
             );
             const eventsPromise = subscribeToEventsWithExtHash(requestIdentifier, context);
 
-            const requestVcCall = await createSignedTrustedCallRequestVc(
-                context.api,
-                context.mrEnclave,
-                context.api.createType('Index', nonce),
-                new PolkadotSigner(context.substrateWallet.alice),
-                aliceSubstrateIdentity,
-                context.api.createType('Assertion', assertion).toHex(),
-                context.api.createType('Option<RequestAesKey>', aesKey).toHex(),
-                requestIdentifier
-            );
+            let requestVcCall;
+            if (Array.isArray(assertion)) {
```

Comment:
The di and dr vc test codes are basically the same, so you may need to change the code of di to be the same as dr's, otherwise I think ci should not pass.  cc @BillyWooo 

Since they only have a difference between isVcDirect true and false, do we need to consider merging them? Otherwise, in the future changing the same code will look too bloated. cc @Traf333

Reply:
Let's visit this topic together with **Whether we should remove `request_vc`, and only use `request_batch_vc` in the future?**. Then we maybe do some test refactor together.